great_expectations/checkpoint/checkpoint.py

```diff
@@ -632,7 +634,7 @@ def _run_default_validation_operator(
         evaluation_parameters: Optional[dict] = None,
         run_name: Optional[str] = None,
         run_time: Optional[Union[str, datetime.datetime]] = None,
-        result_format: Union[str, dict] = {"result_format": "SUMMARY"},
+        result_format: Optional[Union[str, dict]] = None,
```

Comment:
why are you adding optional here and removing defaults, only to set them again in the method body?

Reply:
@donaldheppner This is important: putting defaults in the arguments is a mutability hazard.  We must avoid this practice.  Thanks.